phpcs.xml.dist

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards">
+	<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress-Core/ruleset.xml -->
+
+	<description>WooCommerce dev PHP_CodeSniffer ruleset.</description>
+
+	<!-- Exclude paths -->
+	<exclude-pattern>*/node_modules/*</exclude-pattern>
+	<exclude-pattern>*/vendor/*</exclude-pattern>
+	<exclude-pattern>bin/*</exclude-pattern>
+
+	<!-- Configs -->
+	<config name="minimum_supported_wp_version" value="4.7" />
+	<config name="testVersion" value="5.6-"/>
+
+	<!-- Rules -->
+	<rule ref="WooCommerce-Core" />
+
+	<rule ref="WordPress.WP.I18n">
+		<properties>
+			<property name="text_domain" type="array" value="woocommerce-admin" />
+		</properties>
+	</rule>
+
+	<rule ref="PHPCompatibility">
+		<exclude-pattern>tests/</exclude-pattern>
+	</rule>
+
+	<rule ref="WordPress.Files.FileName.InvalidClassFileName">
+		<exclude-pattern>src/*</exclude-pattern>
```

Comment:
Looks like some of these rules need to be update or removed because WCS does not have a src/ directory. 
I think the InvalidClassFileName rule doesn't need any exclusions except maybe the tests/ folder.
For the other rules I think src/ should be replaced with classes/

Reply:
Good point. Confirmed InvalidClassFileName doesn't apply to our `classes` folder since all ours have `class-` prefix
https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/WordPress/Sniffs/Files/FileNameSniff.php#L190-L191

Reply:
> InvalidClassFileName

Ended up our test classes have `_` instead of `-`. We can probably rename all of them.